web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1713,6 +1713,8 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   @volatile var cometGetTimeout = 140000
 
+  @volatile var putAjaxFnsInContainerSession = false
```

Comment:
That is a fair point. I wish we had a _set at most once_ semantics thingy lying around. How have we handled this with other settings?